Blog/include/functions.php

```diff
@@ -54,6 +54,26 @@ function printHeader(){
 		echo "<html>";
 };
 
+function getAllTags(){
+	$result = dbQuery("
+		SELECT *
+		FROM Tags
+	")->fetchAll();
+
+	return $result;
+}
+
+function getTagBlogs($tagID){
+	$result = dbQuery("
+		SELECT *
+		FROM PostTag
+		WHERE tagID = :tagID
+		",
+	 array('tagID' => $tagID )
+ );
```

Comment:
style suggestion: I would align this line 73 with line 67. 
Wherever something is opened it should close in the same relative position 
~function getTagBlogs($tagID){
	$result = dbQuery("
		SELECT *
		FROM PostTag
		WHERE tagID = :tagID
		",
	 array('tagID' => $tagID )
        );
          return $result -> fetchAll();
} ~